drools-wb-services/drools-wb-verifier/drools-wb-verifier-webworker/pom.xml

```diff
@@ -77,6 +77,7 @@
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>gwt-maven-plugin</artifactId>
         <configuration>
+          <skip>false</skip>
```

Comment:
This does do the trick, but not sure if this is the best way. For some reason drools-wb-showcase runs by GWT by default, but there is no easy ( for me ) to spot reason why the webworker should not do the same. Spending some more time on investigating this.